jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Where is the api for fetching graviton usage ?

Reply:
The payer account activated the tag: architecture.
By using this tag we can get the cost of graviton instance usage.